models/issue.go

```diff
@@ -1256,7 +1263,15 @@ func Issues(opts *IssuesOptions) ([]*Issue, error) {
 	if err := IssueList(issues).LoadAttributes(); err != nil {
 		return nil, fmt.Errorf("LoadAttributes: %v", err)
 	}
-
+	if opts.RenderEmojiTitle == util.OptionalBoolTrue {
+		var issuesWithEmojis []*Issue
+		for _, issue := range issues {
+			title := string(markup.RenderEmoji(issue.Title))
+			issue.Title = title
+			issuesWithEmojis = append(issuesWithEmojis, issue)
+		}
+		return issuesWithEmojis, nil
+	}
```

Comment:
Rendering of the emoji should be done elsewhere.

Reply:
Why was it made an option? I thought emoji rendering to be a non-destructive operation that should always be performed where possible.

Reply:
the search API is being used to display issue in the board search and you requested it shows emojis. Where else could this code go? In the JavaScript? Sounds not practical to me.

Reply:
You can probably use `markup.RenderEmoji`, probably in `routers/api`.

Reply:
@silverwind this would require every function in api that queries a list of issues. It makes more sense to leave it the central Issues function so that all can benefit.

Reply:
It should move elsewhere or use one of the shared functions. Thematically, a issue has nothing to do with emoji rendering, we're doing that all over the place, not just in issues.